lib/miq/lwimiq.rb

```diff
@@ -98,5 +103,13 @@ def uri_for(repo)
         ].join("&")
       )
     end
+
+    def file_rename
+      @subtitle == '<subtitle>' ? 'CHANGE-ME' : cleanup
+    end
+
+    def cleanup
```

Comment:
WDYT about "slug" for this?

Reply:
Do you mean to rename the method to `slug`?